core/src/translate.rs

```diff
@@ -380,10 +441,15 @@ pub fn translate_assignment(sql_assignment: &SqlAssignment) -> Result<Assignment
             .ok_or(TranslateError::UnreachableEmptyIdent)?
             .value
             .to_owned(),
-        value: translate_expr(value)?,
+        value: translate_expr_with_params(value, params)?,
     })
 }
 
+pub fn translate_assignment(sql_assignment: &SqlAssignment) -> Result<Assignment> {
```

Comment:
It would be great to add a short doc comment here with a `# Errors` section explaining when this `Result` can fail.